src/lib.rs

```diff
@@ -48,7 +48,7 @@ extern crate bincode;
 extern crate config_file_handler;
 #[macro_use]
 extern crate lazy_static;
-#[macro_use]
+#[cfg_attr(test, macro_use)]
```

Comment:
Yeah, I've been trying out Rust's new incremental compilation, and the new compiler is smart enough to know that the macros are only used in tests.